client/a8c-for-agencies/sections/partner-directory/dashboard/index.tsx

```diff
@@ -294,11 +294,8 @@ const PartnerDirectoryDashboard = () => {
 			<div className="partner-directory-dashboard__completed-section">
 				<div className="partner-directory-dashboard__heading">
 					{ translate(
-						'Thank you! You’ll be notified when the partner directory is live.',
-						'Thank you! You’ll be notified when the partner directories are live.',
-						// todo: Once the partner directory are live use the copy below:
```

Comment:
haha. I love that you (or someone) left comments for future changes. professional @vitozev

Reply:
> // todo: Once the partner directory are live use the copy below:

That was the comment I left in... July ( wow...how time flies... )